egs/wsj/s5/steps/nnet3/report/generate_plots.py

```diff
@@ -52,9 +52,6 @@ def get_args():
         --comparison-dir exp/nnet3/tdnn1 --comparison-dir exp/nnet3/tdnn2 \\
         exp/nnet3/tdnn exp/nnet3/tdnn/report""")
 
-    parser.add_argument("--comparison-dir", type=str, action='append',
```

Comment:
It needs to be back-comptiable to the old usage method, but you can mention that this option is deprecated.